src/controller/model/endpoint.ts

```diff
@@ -714,10 +714,10 @@ class Endpoint extends Entity {
             optionsWithDefaults.reservedBits,
         );
 
-        const log =
+        const messageBuilder = (): string =>
             `CommandResponse ${this.deviceIeeeAddress}/${this.ID} ` +
             `${cluster.name}.${command.name}(${JSON.stringify(payload)}, ${JSON.stringify(optionsWithDefaults)})`;
-        logger.debug(log, NS);
+        logger.debug(() => messageBuilder(), NS);
```

Comment:
No wrapping anon function necessary here, can just pass `messageBuilder` as param. Same for all other occurrences in the PR.

Reply:
Done.

Reply:
@Koenkk Your opinion on the name for these functions? `messageBuilder` doesn't follow the usual "action/verb" pattern of a function. `getLogString`?

Reply:
What about `createLogMessage`?